### DISCUSSION TITLE
Support of Virtualisation for M2 IPad Air?

### DISCUSSION THREAD

**Discussion by amongussussybaka228** [utmapp/UTM#6922] [UTM Help (iOS)]
Will virtualisation be available for M2 IPad Air? JIT can be enabled

* UTM Version: 4.6.4
* OS Version: IOS 18.2
* Device Model: IPad Air 2024 (M2)
* Not jailbroken with JIT enabled 
* Sideloading from GitHub 

Sorry for short question xD

**Comment by osy** [comment]
https://docs.getutm.app/installation/ios/#trollstore
